docker-compose.yml

```diff
@@ -76,6 +76,19 @@ services:
         "--smtp-auth-allow-insecure=true"
       ]
 
+  # SMTP server with TLS
+  smtp4dev:
```

Comment:
Is this only needed for testing? If so, can you call it `smtp4dev_test`

Reply:
good idea, done